src/schoology-extractor/schoology_extractor/mapping/discussion_replies.py

```diff
@@ -0,0 +1,86 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+from datetime import datetime
+import pandas as pd
+
+from . import constants
+
+DISCUSSION_REPLIES_TYPE = "Discussion Reply"
+
+
+def map_to_udm(discussion_replies_df: pd.DataFrame, section_id: int) -> pd.DataFrame:
+    """
+    Maps a DataFrame containing Schoology section associations (enrollments)
+    into the Ed-Fi LMS Unified Data Model (UDM) format.
+
+    Parameters
+    ----------
+    enrollments_df: DataFrame
+        Pandas DataFrame containing Schoology assignments for a section
+
+    Returns
+    -------
+    DataFrame
+        A LMSUsers-formatted DataFrame
+
+    Notes
+    -----
+    DataFrame columns are:
+        SourceSystemIdentifier: A unique number or alphanumeric code assigned to a the discussion-reply by
+            the source system
+        SourceSystem: The system code or name providing the user data
+        LMSUserIdentifier: A unique number or alphanumeric code assigned to a user by the source
+            system
+        LMSSectionIdentifier: A unique number or alphanumeric code assigned to a section by the
+            source system
+        EntityStatus: The status of the record
+        ActivityDateTime: The date/time the replied was created.
+        ActivityStatus: The status for the reply
+        ActivityType: The type of activity: `Discussion reply`
+        Content: The comment text.
+        LMSUserActivityIdentifier: A unique numeric identifier assigned to the user activity:
```

Comment:
This is an unnecessary column. The UDM shows it, but it would only appear in the RDBMS as an auto-incrementing integer. Please remove. We don't have a column to place the `discussion_id` into.

Reply:
✔️